### ISSUE TITLE
Add background map option that places streetnames next to rather than on the road. 

### ISSUE THREAD

**Issue by InsertUser** [Zaczero/osm-relatify#74]
The current Relatify interface covers the roads in the background map with an overlay showing the route as selected and branching streets that can also be added to the route. This means that the names of the streets are covered up if they are displayed on the road. This can make it difficult to see if you are selecting the correct road in some instances. 

Other than the option described in #45 this could be solved by rendering the road names on top of the overlay, which might hide important information or by choosing a background layer that draws the road names next to the roads rather than in them. The latter option is probably the least effort by far. 

Potential background layers with pros (➕) and cons (➖):
- Cycle map (Thunderforest), 
-- ➕displays roads like this for everything below secondary 
-- ➖secondary and higher roads have label in the way
-- ➖would probably need an API key or special permission
- Transport map (Thunderforest), 
-- ➕displays road names alongside roads for all ways 
-- ➕shows existing bus routes, so could be used to assist the planned route selection mechanism (to guide a "show nearby routes" query)
-- ➖would probably need an API key or special permission
- OSM Americana (ZeLonewolf)
-- ➖ needs vector tile support (although there is a leaflet plugin for OpenMapTile [apparently][1])
-- ➖ Relatify domain [would need][2] to be whitelisted for use. 
-- ➕ shows labels "alongside" for road below secondary
-- ➖ by default shows labels within the road for roads above secondary ... but as this is a vector map style this might be fixable with a "light" fork?

[1]: https://leafletjs.com/plugins.html#vector-tiles
[2]: https://github.com/ZeLonewolf/openstreetmap-americana/blob/main/TILE_USAGE.md

**Comment by Zaczero** [comment]
We could also make the lines slightly transparent:

![image](https://github.com/Zaczero/osm-relatify/assets/10835147/e4f85f2a-3563-4616-beef-73c65d3819ad)

![image](https://github.com/Zaczero/osm-relatify/assets/10835147/734b9a84-3e21-4eb2-9bc9-514d7e2c7f82)

I tested this also on the orange lines, but the names were still unreadable (due to the line animating)

**Comment by mutipg** [comment]
Good idea with the transparent gray lines. This will be very helpful in route edits!
